lib/themes/dosomething/paraneue_dosomething/js/campaign/prototype.js

```diff
@@ -58,23 +57,16 @@ $(document).on('click', '.js-kudos-button', function(event) {
   const dataKid = $el.attr('data-kid');
   const method = dataKid ? 'DELETE' : 'POST';
 
-  // if (setting('dsKudosReactions.sendToRogue')) {
-  //   makeKudosRequestToRogue(setting('dsKudosReactions.northstarId'), setting('dsKudosReactions.postId'))
-  // } else {
   makeKudosRequest($el, dataKid)
     .done((response) => {
       const kid = get(response, '0.kid', '');
-
-      if (method === 'POST' && !kid) {
-        return;
-      }
-
+      // if (method === 'POST' && !kid) {
+      //   return;
+      // }
```

Comment:
Right! The `POST` endpoint [return a `kid`](https://github.com/DoSomething/phoenix/blob/b43bc156041a59dcb08894df4132ad93065be3e1/lib/modules/dosomething/dosomething_api/resources/kudos_resource.inc#L203-L207) if it's successful (which we're using the `post_id` as a stand-in for). The idea behind this conditional is that if we're posting a new kudo but a value isn't returned in `0.kid` then we assume an error.

Can you try logging out the `response` variable above? Maybe it's not structured how we expect it to be here?